test/common/tests.js

```diff
@@ -3,6 +3,7 @@
 const assert = require('assert')
 const stream = require('stream')
 const { join } = require('path')
+const util = require('util')
```

Comment:
can we pull just `format` out if that's all we are using (`const { format } = require('util');`)